pkg/attestlib/pkix_test.go

```diff
@@ -124,3 +124,22 @@ func TestCreatePkixAttestation(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Doesn't have to be in this PR: can you have some similar end-to-end tests for PKIX as in https://github.com/grafeas/kritis/blob/master/pkg/attestlib/e2etest/pgp_e2e_test.go?

Reply:
Acknowledged. I will send out another PR for the end to end test.